sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuJsonTuple.scala

```diff
@@ -70,8 +76,8 @@ case class GpuJsonTuple(children: Seq[Expression]) extends GpuGenerator
         }
       }
       withResource(fieldScalars) { fieldScalars =>
-        withResource(fieldScalars.safeMap(field => json.getJSONObject(field))) { resColumns =>
-          withResource(new Table(resColumns: _*)) { tbl =>
+        withResource(fieldScalars.safeMap(field => json.getJSONObject(field))) { resCols =>
+          withResource(new Table((notJsonTupleCols.toArray ++ resCols): _*)) { tbl =>
             GpuColumnVector.from(tbl, schema)
           }
```

Comment:
Nit: It's a bit wasteful to create a `Table` only to immediately throw it away.
```suggestion
          val batchCols = (notJsonTupleCols ++ resCols).zip(schema).map { case (col, dataType) =>
            GpuColumnVector.from(col, dataType)
          }
          new ColumnarBatch(batchCols.toArray, batchCols(0).getRowCount)
```

Although it would be even better to avoid pulling apart the notJsonTupleCols just to put them back together again.  Those already start out as GpuColumnVector instances, and we should just be able to increfcount directly on those and pass them through without going to the cudf col and back to a new GpuColumnVector instance.